02-Workflow-Overview.Rmd

```diff
@@ -1,11 +1,165 @@
 # Workflow Overview
 
-## Starting a new project
+Tasks:
+
+1.  If you are not already a member of the [ROSSyndicate GitHub
+Organization](https://github.com/rossyndicate), please Teams Katie Willi to add
+you.
+
+## Starting a new repository
+
+1.  From the ROSSyndicate Organization landing page, create a new repository
```

Comment:
note: this looks weird on the web and I hate it, but it renders fine. Basically when I knit, it just collapses all the bullets into a single para. Really strange and frustrating for making things 'literate'.